src/test/resources/repositories/bibtex.ecore

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Yeah, of course. I just used `/resources` as that's the Java convention for files used externally. I think a rename is smart, though since `fixtures` is more accurate than `resources`.